.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+cache:
+ directories:
+ - node_modules
+notifications:
+ email: false
+node_js:
+- '7'
```

Comment:
That means that the CI will be executed using node versions 7 and 8, choose only one version, that is helpful when you are develop a nodejs lib.